### PR TITLE
PORT-7464-move-webhook-integration-examples-to-integrate-with-any-tool

### PR DESCRIPTION
**Pull request by kodjomiles**
# Description

moved dynatrace to intergrate port with any tool

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- build-your-software-catalog/sync-data-to-catalog/apm-alerting/dynatrace

image

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/apm-alerting/dynatrace.md [hunk 1]
```diff
@@ -1211,4 +1215,260 @@ The combination of the sample payload and the Ocean configuration generates the
...
+:::
+
+Done! any problem detected on your Dynatrace entity will trigger a webhook event. Port will parse the events according to the mapping and update the catalog entities accordingly.
+
+### Ingest Dynatrace Entities
+
+In this example you will create a `dynatrace_entity` blueprint that ingests monitored entities from your Dynatrace account. You will then add some python script to make API calls to Dynatrace REST API and fetch data for your account.
+
+- [Code Repository Example](https://github.com/port-labs/example-dynatrace-entities)
+
```
Comment (PeyGis) [inline docs/build-your-software-catalog/sync-data-to-catalog/apm-alerting/dynatrace.md:1326]: This example for ingesting entities should come after the webhook installation method (after mapping result)

**Review by PeyGis — CHANGES_REQUESTED** [review]
LGTM

File: docs/build-your-software-catalog/sync-data-to-catalog/apm-alerting/dynatrace.md [hunk 2]
```diff
@@ -1211,4 +1215,273 @@ The combination of the sample payload and the Ocean configuration generates the
   },
 }
 ```
-</details>
\ No newline at end of file
+</details>
+
+To adjust the content as suggested, you'll need to move the section that describes creating the `dynatrace_entity` blueprint and the Python script example to come after the "Mapping Result" section in the documentation.
+
+Here’s how you can revise the flow:
```
Comment (PeyGis) [inline docs/build-your-software-catalog/sync-data-to-catalog/apm-alerting/dynatrace.md:1222]: this is out of context so remove it

**Review by PeyGis — CHANGES_REQUESTED** [review]
LGTM

**Review by PeyGis — APPROVED** [review]
previous comments have been addressed so this looks good to me

File: docs/build-your-software-catalog/sync-data-to-catalog/apm-alerting/datadog.md [hunk 3]
```diff
@@ -1648,6 +1651,187 @@ The combination of the sample payload and the Ocean configuration generates the
 
 </details>
 
+
 ## Alternative installation via webhook
 
-While the Ocean integration described above is the recommended installation method, you may prefer to use a webhook to ingest data from Datadog. If so, follow this [guide](/build-your-software-catalog/custom-integration/webhook/examples/datadog).
+We can create a webhook integration between [Datadog](https://www.datadoghq.com/) and Port, which will ingest alerts and monitor entities into Port and map them to your microservice entities.
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/apm-alerting/datadog.md:1657]: @kodjomiles For consistency, the opening paragraph of this section is the same for all integrations, for example:
https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/project-management/jira/#alternative-installation-via-webhook

Please keep this paragraph in all integrations in this PR

**Review by hadar-co — CHANGES_REQUESTED** [review]

**Review by hadar-co — APPROVED** [review]

**Pull request merged**